app/api/threads/list_threads.feature

```diff
@@ -0,0 +1,172 @@
+Feature: Get threads
+  Background:
+    Given there are the following users:
+      | login               |
+      | PresidentConsortium |
+      | PresidentUniversity |
+      | RichardFeynman      |
+      | Mary                |
+      | EtienneKlein        |
+      | Charlotte           |
+      | Baptiste            |
+      | Thibaut             |
+      | DavidBowie          |
+    And there are the following groups:
+      | parent               | name                 |
+      |                      | UniversityConsortium |
+      | UniversityConsortium | University           |
+      | University           | PresidentUniversity  |
+      | University           | FirstYear            |
+      | FirstYear            | Classroom            |
+      | UniversityConsortium | Université           |
+      | Université           | PremièreAnnée        |
+      | PremièreAnnée        | Classe               |
+      |                      | Superstar            |
+    And there are the following group members:
+      | group                | member              |
+      | UniversityConsortium | PresidentConsortium |
+      | University           | RichardFeynman      |
+      | Classroom            | Mary                |
+      | Université           | EtienneKlein        |
+      | PremièreAnnée        | Charlotte           |
+      | PremièreAnnée        | Baptiste            |
+      | PremièreAnnée        | Thibaut             |
+      | Superstar            | DavidBowie          |
+    And there are the following items with permissions:
```

Comment:
In the same way as "groups", it is items with permissions (a 1-n relationship btw, strange to show it as a table), I would expect the "key" (so item) to be the first column.

Reply:
fixed